0.14.0* reference to react prevents nexus generation

I'm using this in a project, and updated to 0.14.0-next.1 to solve an issue I ran into when I updated Typescript. However, a new error occurred -- dmmf/utils.d.ts line 1 (in the published npm package for 0.14.0-next.1) has a line:
```
/// <reference types="react" />
```
for no apparent reason, and this prevents nexus from generating. If I manually delete the line out of the .d.ts file, it builds fine. I can't tell where or why this gets generated, but it appears to be a bug.

In my case nexus generation works fine - following examples I'm generating schema in transpileOnly mode (`ts-node -T`)

But there is an issue with this reference when trying to compile project - that forces me to use `skipLibcheck:true`

**Repro**

clone some helloworld from ./examples

```
$ tsc
```

```
node_modules/nexus-prisma/dist/dmmf/utils.d.ts:1:23 - error TS2688: Cannot find type definition file for 'react'.

1 /// <reference types="react" />
                        ~~~~~

```

```
{
  "compilerOptions": {
    "target": "es2018",
    "module": "commonjs",
    "lib": ["esnext"],
    "strict": true,
    "rootDir": "src",
    "outDir": "build",

    "esModuleInterop": true
    // "skipLibCheck": true
  },
  "include": ["src"]
}
```

```
 "dependencies": {
    "@nexus/schema": "^0.14.0",
    "@prisma/client": "^2.0.0-beta.8",
    "graphql": "^14.6.0",
    "nexus-prisma": "^0.14.0"
  },
  "devDependencies": {
    "@prisma/cli": "^2.0.0-beta.8",
    "typescript": "^3.9.5"
  }
```

--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
     "release:stable": "dripip stable",
     "release:pr": "dripip pr",
     "clean": "rm -rf dist",
-    "build": "ts-node scripts/ensure-prisma2-version && yarn -s clean && tsc",
+    "build": "ts-node scripts/ensure-prisma2-version && yarn -s clean && tsc && ts-node scripts/post-build",
     "dev": "tsc --watch",
     "dev:test": "ts-node scripts/ensure-prisma2-version && ts-node tests/__ensure-engine && jest --watch",
     "prepublishOnly": "yarn -s build",

--- a/scripts/post-build.ts
+++ b/scripts/post-build.ts
@@ -1,3 +1,7 @@
+/**
+ * This script is a temp fix, hack.
+ * https://github.com/graphql-nexus/nexus-schema-plugin-prisma/pull/700.
+ */
 import * as fs from 'fs-jetpack'
 
 let contents

--- a/scripts/post-build.ts
+++ b/scripts/post-build.ts
@@ -1,0 +1,12 @@
+import * as fs from 'fs-jetpack'
+
+let contents
+let path
+
+path = 'dist/dmmf/utils.d.ts'
+contents = fs.read(path)
+
+if (!contents) throw new Error(`No such path: ${path}`)
+
+contents = contents.replace('/// <reference types="react" />\n', '')
+fs.write(path, contents)

